src/Data/Fin/Induction.agda

```diff
@@ -59,6 +60,20 @@ open WF public using (Acc; acc)
   induct {suc i} (acc rec) = Pᵢ⇒Pᵢ₊₁ i (induct (rec (inject₁ i) i<i+1))
     where i<i+1 = ℕ<⇒inject₁< (i<1+i i)
 
+≤-Induction : ∀ (P : Pred (Fin (suc n)) ℓ) {i} →
+              P i →
+              (∀ {i} → P (inject₁ i) → P (suc i)) →
+              ∀ {j} → j ≥ i → P j
+≤-Induction P {i} Pi Pᵢ⇒Pᵢ₊₁ {j} j≥i = induct (<-wellFounded _) (<-cmp i j) j≥i
+  where
+  induct : ∀ {j} → Acc _<_ j → Tri (i < j) (i ≡ j) (i > j) → j ≥ i → P j
+  induct {suc j} (acc rs) (tri< (s≤s i≤j) _ _) _ = Pᵢ⇒Pᵢ₊₁ finB
+    where
+    finB = induct (rs _ (s≤s (subst (ℕ._≤ toℕ j) (sym $ toℕ-inject₁ j) ≤-refl)))
```

Comment:
`finB` isn't a great variable name. Maybe something like `P[1+j]`? Also could we split the term out into some slightly more meaningful named subparts? For example `sym $ toℕ-inject₁ j` is duplicated twice.

Reply:
I modified the name and created a new one for `sym $ toℕ-inject₁ j`.